mir-ci/pyproject.toml

```diff
@@ -32,3 +32,25 @@ exclude = [
 dependencies = [
     "pywayland @ git+https://github.com/Saviq/pywayland.git@patch-1",
 ]
+
+[tool.black]
+line-length = 120
+
+[tool.isort]
+line_length = 120
+profile = "black"
+
+[tool.mypy]
+warn_return_any = true
+warn_unused_configs = true
+exclude = [
+    "^hatch_build.py",
+]
+
+[[tool.mypy.overrides]]
+module = "inotify.*"
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module = "gi.*"
+ignore_missing_imports = true
```

Comment:
Should we add a `requirements_dev.txt` that includes all of the tools that we use? Then we can install with:
```
pip install -r requirements_dev.txt
```
and get black, isort, mypy, and flake8 without having to do individual installs?

Reply:
Yeah I've not decided what's best… But I'm leaning towards `pytest --deps` doing the dependencies, since we already have that, and `pytest --lint` running `black` and `isort` without `--check`…

I mean, it's one thing getting feedback from linters in CI, another to integrate them into your workflow, pre-commit hook etc.